docs/enforcement/inline.asciidoc

```diff
@@ -0,0 +1,255 @@
+
+Another important setting is the `gateway` statement. Since it this the only way to get the PacketFence server inline interface IP address, it is mandatory to set it to this IP (which is supposed to be the same as in the `ip` statement of the inline interface in `conf/pf.conf`).
+
+Preparating the Operating System
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to build an inline deployment of PacketFence setup you need :
+
+* 2 network interfaces for the VM (1 for the Inline and another one to go out)
+* a switch port in the management network for the PacketFence server
+* a switch port in the inline network for the PacketFence server which needs to be configured in access mode and in the same access VLAN as every switchport on which devices will be connected
+* your server has net.ipv4.ip_forward enabled
+
+Edit the following file:
+----
+# /etc/sysctl.conf
+----
+
+Change net.ipv4.ip_forward from 0 to 1
+----
+# net.ipv4.ip_forward = 1
+----
+
+Now you need to make this change permanent, apply the following in your terminal:
+----
+# sysctl -p /etc/sysctl.conf
+----
+
+Adding Inline Interface
+~~~~~~~~~~~~~~~~~~~~~~~
+
+PacketFence can be configured right from the start using the PacketFence configurator using the inline enforcement. In this example, we will continue building on top or our initial deployment by adding a new inline interface to our PacketFence installation.
+
+The first step is to add a dedicated Network Interface Card (NIC) to your current PacketFence installation. In our example, our new NIC will be named `ens192`. The PacketFence web interface will list all currently installed network interfaces on the system. An IP and a netmask will be visible if the network interface is configured (either by DHCP or already manually configured). You can edit those ones, create/delete VLANs on physical interfaces and enable/disable an interface. Note that theses changes are effective on the moment you make them. Persistence will be written only for *enabled* interfaces. Which means that if you change your management IP address, to pursue the configurator, you will need to go on this new IP address you setup. At all time, you'll need to set a Management interface. That means that the required interface types for inline enforcement are
+
+    Management
+    Inline layer 2
+
+Note that PacketFence will these services on its inline interface:
+
+* PacketFence provides its own DHCP service. It will take care of IP address distribution in our Inline network. PacketFence will not provide DHCP services on the management network - this is the responsibility of your own infrastructure.
+* PacketFence provides its own DNS service. However for the Inline version, we need to provide the DNS of your infrastructure.
```

Comment:
However, for the inline mode, you will also need to provide access to the DNS server of your infrastructure